cmd/plugin/rpaasv2/cmd/info.go

```diff
@@ -0,0 +1,230 @@
+// Copyright 2020 tsuru authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package cmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"strconv"
+	"text/template"
+
+	"github.com/olekukonko/tablewriter"
+	"github.com/tsuru/rpaas-operator/pkg/apis/extensions/v1alpha1"
+	rpaasclient "github.com/tsuru/rpaas-operator/pkg/rpaas/client"
+	clientTypes "github.com/tsuru/rpaas-operator/pkg/rpaas/client/types"
+	"github.com/urfave/cli/v2"
+)
+
+func NewCmdInfo() *cli.Command {
+	return &cli.Command{
+		Name:  "info",
+		Usage: "Retrieves information of the rpaas-operator instance given",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:    "service",
+				Aliases: []string{"tsuru-service", "s"},
+				Usage:   "the Tsuru service name",
+			},
+			&cli.StringFlag{
+				Name:     "instance",
+				Aliases:  []string{"tsuru-service-instance", "i"},
+				Usage:    "the reverse proxy instance name",
+				Required: true,
+			},
+			&cli.BoolFlag{
+				Name:    "raw-output",
+				Aliases: []string{"r"},
+				Usage:   "show as JSON instead of go template format",
+				Value:   false,
+			},
+		},
+		Before: setupClient,
+		Action: runInfo,
+	}
+}
+
+func prepareTemplate() (*template.Template, error) {
+	tmp := `
+{{- with .Name }}
+Name: {{ . }}
+{{- end }}
+{{- with .Team }}
+Team: {{ . }}
+{{- end }}
+{{- with .Description }}
+Description: {{ . }}
+{{- end }}
+{{- with .Replicas }}
+Replicas: {{ . }}
+{{- end }}
+{{- with .Plan }}
+Plan: {{ . }}
+{{- end }}
+{{- with .Binds }}
+
+Binds:
+{{ formatBinds . }}
+{{- end }}
+{{- with .Tags }}
+Tags:
+{{ formatTags . }}
+{{- end }}
```

Comment:
Same as above, we always should show the tags; and consider move it to next to the fixed fields' section (as suggested above).